packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.integration.test.tsx

```diff
@@ -47,7 +48,7 @@ const mockUseGetCurrentUserBundle = useGetCurrentUserBundle as jest.Mock
 
 const mockToastFn = jest
   .spyOn(ToastMessage, 'displayToast')
-  .mockImplementation(() => {})
+  .mockImplementation(() => noop)
```

Comment:
Small change to `displayToast` to return a function that closes it. Updated mocks to satisfy TypeScript